galactic/distribution.yaml

```diff
@@ -2015,6 +2015,17 @@ repositories:
       url: https://github.com/osrf/nodl_to_policy.git
       version: master
     status: maintained
+  nonpersistent_voxel_layer:
+    release:
+      tags:
+        release: release/galactic/{package}/{version}
+      url: https://github.com/SteveMacenski/nonpersistent_voxel_layer-release.git
+      version: 2.2.2-1
+    source:
+      type: git
+      url: https://github.com/SteveMacenski/nonpersistent_voxel_layer.git
```

Comment:
Please add a LICENSE file to the repository. Thank you kindly!

Reply:
Oh huh, that was an oversight. https://github.com/SteveMacenski/nonpersistent_voxel_layer/blob/ros2/LICENSE now there, will backport to galactic

Reply:
https://github.com/SteveMacenski/nonpersistent_voxel_layer/blob/galactic/LICENSE